sdk/communication/azure-communication-networktraversal/src/main/java/com/azure/communication/networktraversal/CommunicationRelayAsyncClient.java

```diff
@@ -49,11 +60,34 @@
     /**
      * Creates a new CommunicationRelayConfiguration.
```

Comment:
Is this method "get" or "create"? If it is indeed `get`, can you update all the javadocs?

Reply:
Get, but in the previous iteration was changed.